addons/website/models/ir_ui_view.py

```diff
@@ -293,6 +295,11 @@ def customize_template_get(self, cr, uid, key, full=False, bundles=False, contex
                 })
         return result
 
+    def toggle_options(self, cr, uid, ids, context=None):
+        view_ids = self.search(cr, uid, [('inherit_id', 'child_of', ids), ('active', '=', True)], context=context)
+        ids = view_ids if view_ids and len(view_ids) else ids
+        self.toggle(cr, uid, ids, context=context)
```

Comment:
remove the line 300 and use self.toggle(cr, uid, view_ids or ids, context=context)